install_config/install/advanced_install.adoc

```diff
@@ -209,6 +209,15 @@ agent.
 |If `*ansible_ssh_user*` is not *root*, this variable must be set to *true* and
 the user must be configured for passwordless *sudo*.
 
+|`*debug_level*`
+v|This variable sets which INFO messages are logged to the `systemd-journald.service`. Set one of the following:
+* `0` to log errors and warnings only
+* `2` to log normal information (This is the default level.)
+* `4` to log debugging-level information 
+* `6` to log API-level debugging information (request / response)
+* `8` to log body-level API debugging information
+For more information on debug log levels, see xref:../../install_config/master_node_configuration#config-logging-levels[Configuring Logging Levels]
```

Comment:
The xref needs a .adoc in it (or the Customer Portal build will fail), and a period at the end of the sentence.